frontend/src/App.css

```diff
@@ -1,38 +1,22 @@
 .App {
-  text-align: center;
+    text-align: center;
```

Comment:
Am Anfang der Zeile ist eine Leerzeichen eingefügt, das sicher keine Absicht ist? :) . Hast du vielleicht Autoformat vor Upload vielleicht in Intellij nicht eingestellt?